lib/displayutils.js

```diff
@@ -16,7 +16,18 @@ function resultDisplay(id, prefix, result) {
   }
 
   var line = parts.join(' - ');
-  return (result.skipped ? 'skip ' : (result.passed ? 'ok ' : 'not ok ')) + id + ' ' + line;
+  var status;
+  if (result.skipped) {
+    status = 'skip ';
+  } else if (result.passed && !result.todo) {
+    status = 'ok ';
+  } else if (!result.passed && result.todo) {
+    status = 'todo ';
```

Comment:
@trentmwillis - Right, I agree, but we have been emitting `skip` for skipped, so I don't think we are truly following TAP...

I'm open to fixing (and moving `skip` to `# SKIP` and `todo ` to `# TODO` per  the spec).